automation/test.sh

```diff
@@ -57,6 +58,7 @@ elif [[ $TARGET =~ sig-compute-migrations ]]; then
   export KUBEVIRT_PROVIDER=${TARGET/-sig-compute-migrations/}
   export KUBEVIRT_WITH_CNAO=true
   export KUBEVIRT_NUM_SECONDARY_NICS=1
+  export KUBEVIRT_DEPLOY_NFS_CSI=true
```

Comment:
I now see that we are mixing `KUBEVIRT_STORAGE` into [focus settings](https://github.com/kubevirt/kubevirt/blob/main/automation/test.sh#L374). Would you mind moving it here? (we can of course do it in follow up)

Reply:
Wouldn't this already be too late? (cluster-up already happened)

Reply:
I was referring to this https://github.com/kubevirt/kubevirt/blob/main/automation/test.sh#L374. I think it's too late to have it there. Not sure how it is working. @dhiller any idea?

Reply:
@xpivarc you are absolutely right, L374 will not have any effect IMHO, as it's already after `make cluster-up`. We should move it here.

Reply:
Either in a followup PR or creating an issue would both be fine with me.

Reply:
After looking at `KUBEVIRT_STORAGE` occurrences, I believe `export KUBEVIRT_STORAGE="rook-ceph"` can be safely removed here.
The migration job will have rook-ceph-default exported already and we match on wildcard in https://github.com/kubevirt/kubevirt/blob/ddc8e8ffbdada25b350580a403143e8932bdd9de/hack/functests.sh#L38-L40